test/CryptoToFiat_test.js

```diff
@@ -0,0 +1,244 @@
+        assert.equal(newOwner, owner)
+    });
+
+    it ("Only Owner Update Price Test", async function()
+    { 
+        var price = {
+            "USD": 10,
+            "EURO": 8,
+            "ETH": 0.001
+        }
+        // Check for the Price Update by Non Owner
+        testErrorRevert( cryptoToFiat.updatePrice(JSON.stringify(price), {from:accounts[0]}));
+
+    });
+
+    it ("Only Owner can change the contract owner Test", async function()
+    { 
+        var newOwner = accounts[0];
+        // Update the Owner called by non-owner
+        testErrorRevert(cryptoToFiat.updateOwner(newOwner, {from:accounts[0]}));
+    });
+
+    it ("Update configuration Test Case", async function()
+    { 
+        // Current Owner is accounts[1]
+        let _minBalance = 1000
+        let _conversionUpperLimit = 100
+        let _txnLimitInBlocks = 9
+        let _authorizer = accounts[9];
+
+        await cryptoToFiat.setConfigurations(_authorizer, _minBalance, _conversionUpperLimit, _txnLimitInBlocks, {from:accounts[1]});
+        
+        let authorizer = await cryptoToFiat.authorizer.call();
+        let minBalance = await cryptoToFiat.minBalance.call();
+        let conversionUpperLimit = await cryptoToFiat.conversionUpperLimit.call();
+        let txnLimitInBlocks = await cryptoToFiat.txnLimitInBlocks.call();
+
+        assert.equal(_authorizer, authorizer)
+        assert.equal(_minBalance, minBalance);
+        assert.equal(_conversionUpperLimit, conversionUpperLimit);
+        assert.equal(_txnLimitInBlocks, txnLimitInBlocks);
+    });
+
+    it ("Update configuration Test Case Only By Owner", async function()
+    { 
+        // Current Owner is accounts[1]
+        let _minBalance = 1000
+        let _conversionUpperLimit = 100
+        let _txnLimitInBlocks = 9
+        testErrorRevert(cryptoToFiat.setConfigurations(_minBalance, _conversionUpperLimit, _txnLimitInBlocks, {from:accounts[0]}));
+
+    });
+
+    it ("Initiate Conversion Test Cases", async function()
+    { 
+        let _conversionUpperLimit = 100
+        let depositAmount = 1200
+        var sender = accounts[5]
+        var totalClaim = 10000
+        var lastTxnBlockNumber = 0
+        await token.transfer(accounts[5], depositAmount, {from:accounts[0]});
+        await token.approve(cryptoToFiat.address, depositAmount , {from:accounts[5]}); 
+        await cryptoToFiat.deposit(depositAmount, {from:accounts[5]});
+
+        assert.equal((await cryptoToFiat.balances.call(accounts[5])).toNumber(), depositAmount)
+
+        //Create the signature
+        sender = accounts[5]
+        lastTxnBlockNumber = (await cryptoToFiat.lastTxnBlocks.call(sender)).toNumber();
+
+        let authorizer = await cryptoToFiat.authorizer.call();
+        let sgn = await signFuns.waitSignedInitiateConversionMessage(accounts[9], cryptoToFiat.address, sender, totalClaim, lastTxnBlockNumber);        
+        let vrs = signFuns.getVRSFromSignature(sgn.toString("hex"));
```

Comment:
Are these lines needed? Their result is thrown away by next two lines.

Reply:
Add in other places just as consistency in test cases. I can clean it up.

Reply:
I just cannot understand the point. From the comment above it looks like these lines should test signature creation process but there is no assert on it. We can leave it as is it is minor.